Gemfile.lock

```diff
@@ -297,7 +297,7 @@ DEPENDENCIES
   passenger
   popper_js (~> 1.11, >= 1.11.1)
   pry-rails (~> 0.3.9)
-  rails (~> 5.0)
+  rails (~> 5.2.3)
   rails-perftest
```

Comment:
We're going straight to 5.2? @alaxalves ?

Reply:
Yes, it was a nice suggestion of @IgorWilbert. Since the code is small I think it's a good option we jump straight to 5.2.3